src/Pages/Income.cshtml

```diff
@@ -143,7 +143,7 @@ else
                                         </div>
                                         <div class="col-12 col-md-5 col-lg-4">
                                             <span class="text-info">Amount:</span>
-                                            @NumberFormatter.FormatCurrency(item.Amount, Model.CurrencySymbol)
+                                            @NumberFormatter.FormatCurrency(item.Amount, Model.CurrencyException? item.Currency : Model.CurrencySymbol)
```

Comment:
How exactly did this alter the display when currency is down? And maybe this should be used in for example Saving suggestions, since there are amounts of entries as well.

Reply:
So this works by checking if there was an currency exception, if there was - it displays the currency code of the entry itself, otherwise, in the normal situation, it will display in the currency the user has currently selected

Reply:
But if I'm not mistaken, the calculator will not run, if we can't convert the currencies, so there isn't really any point to doing any changes since they won't be displayed anyway

Reply:
You're right on this one